src/components/events/BreakoutRoomMatchmaker.tsx

```
@@ -0,0 +1,243 @@
+
+export const BreakoutRoomMatchmaker: React.FC<BreakoutRoomMatchmakerProps> = ({
+  eventId = "evt-virtual-1",
+  eventName = "Virtual Networking Mixer 2026",
+  attendees = MOCK_ACTIVE_ATTENDEES,
+  onRoomsGenerated,
+  className,
+}) => {
+  const [targetRoomSize, setTargetRoomSize] = useState<number>(3);
+  const [rooms, setRooms] = useState<BreakoutRoom[]>(() =>
+    matchBreakoutRooms(attendees, 3)
+  );
```

ESLint

[error] Replace `⏎····matchBreakoutRooms(attendees,·3)⏎··` with `·matchBreakoutRooms(attendees,·3)`

(prettier/prettier)